### DISCUSSION TITLE
Transfer personal vault into a collection?

### DISCUSSION THREAD

**Discussion by alexanderharm** [dani-garcia/vaultwarden#2414] [Q&A] [answered]
Hi community, we were wondering if it would be possible to transfer all items of personal vaults into individual collections on database level? We would then proceed to disable the personal vaults. Could this be done via SQL?

**Comment by jjlin** [comment, marked as answer]
No, items moved into an organization/collection use a different encryption key than personal items.

The easiest way is probably to have each user log into the web vault and "Select All" in their personal vault, then "Move Selected to Organization" and choose the appropriate collection.

**Reply by alexanderharm** [reply]
OK, thanks for your input

**Reply by alexanderharm** [reply]
Quick question though: how could one select only the keys of the personal vault? I found no possibility to do this.

**Reply by jjlin** [reply]
I don't know of a way to do that, but if you try to move items already in an org to another org, it should ignore those items.

**Reply by alexanderharm** [reply]
But they would end up in the new collection wouldn't they?

**Reply by jjlin** [reply]
It should not take any action for items already in an org, but you can verify for yourself.
